cypress/integration/ViewTermsByLetter.feature

```diff
@@ -0,0 +1,25 @@
+Feature: View Terms By Letter
+
```

Comment:
It is missing one scenario where there is no results under expanded letter - even though there is no real example, there is a possibility that this could be the case in the future (as always). Could you add the following from AC:
```
 Scenario: User clicks on a letter in the A-Z list for results and gets no matches found for the selected letter
Given the user navigates to "expand/z" (of whatever letter you prefer to mock with no results)
Then search results page displays "No matches were found for your selected letter. Please try a new search, or click a different letter in the alphabet and browse through the list of terms that begin with that letter."
```

Reply:
Catered to this missing scenario now.